static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -285,3 +286,34 @@ export function getCompoundDiseaseContraindication(
   processedData.sort((a, b) => (a.drugSource > b.drugSource ? 1 : -1));
   return processedData;
 }
+
+/**
+ * Fetches the common name of the disease of interest
+ * @param data
+ * @returns - string with disease common name
+ */
+export function getDiseaseCommonName(data: GraphNodes): string {
```

Comment:
Please add a unit test for this exported function

Reply:
Done!